lib/tasks/webpacker/install.rake

```diff
@@ -3,6 +3,13 @@ WEBPACKER_APP_TEMPLATE_PATH = File.expand_path('../../install/template.rb', __di
 namespace :webpacker do
   desc "Install webpacker in this application"
   task :install do
+    # Calling exec before invoking app:tempalte will not proceed
+    # exec 'yarn --version'
+    # Checking yarn binstub - will not work because yarn binstub will not exist yet
+    # `./bin/yarn --version`
+    # Check yarn global - could work, but we are already depning on the yarn binstub,
+    # so why not utilize it as the central point truth for existence of yarn global?
+    # `yarn --version`
     if Rails::VERSION::MAJOR >= 5
       exec "./bin/rails app:template LOCATION=#{WEBPACKER_APP_TEMPLATE_PATH}"
     else
```

Comment:
@gauravtiwari Actually, I take that back.  The yarn binstub is created with `rails new` when using Rails 5.1.0 -- with or without the `webpack` option.  I think `./bin/yarn --version` might actually work here to check that yarn is installed prior to running invoking the `rails app:template`.  I can keep testing to see if that way works out.